src/lando/pushlog/pushlog.py

```diff
@@ -21,46 +21,49 @@ def PushLogForRepo(repo: Repo, user: str):
     WARNING: Do not use record_push() on the returned PushLog, as the context manager
     will take care of it automatically. Calling it separately may lead to duplicates.
```

Comment:
Wondering if there's a way to detect/enforce this.

Reply:
Ah, yep! I added a boolean to record the recording, and I can throw a RuntimeException if we do it twice.